source/MaterialXContrib/MaterialXNode/MaterialXNode.cpp

```diff
@@ -32,19 +32,19 @@
 const MTypeId MaterialXNode::MATERIALX_NODE_TYPEID(0x00042402);
 const MString MaterialXNode::MATERIALX_NODE_TYPENAME("MaterialXNode");
 
-MString MaterialXNode::DOCUMENT_ATTRIBUTE_LONG_NAME("document");
+MString MaterialXNode::DOCUMENT_ATTRIBUTE_LONG_NAME("documentFilePath");
 MString MaterialXNode::DOCUMENT_ATTRIBUTE_SHORT_NAME("doc");
 MObject MaterialXNode::DOCUMENT_ATTRIBUTE;
 
-MString MaterialXNode::ELEMENT_ATTRIBUTE_LONG_NAME("element");
+MString MaterialXNode::ELEMENT_ATTRIBUTE_LONG_NAME("elementPath");
 MString MaterialXNode::ELEMENT_ATTRIBUTE_SHORT_NAME("ele");
 MObject MaterialXNode::ELEMENT_ATTRIBUTE;
 
 const MTypeId MaterialXTextureNode::MATERIALX_TEXTURE_NODE_TYPEID(0x00042403);
-const MString MaterialXTextureNode::MATERIALX_TEXTURE_NODE_TYPENAME("MaterialXTextureNode");
+const MString MaterialXTextureNode::MATERIALX_TEXTURE_NODE_TYPENAME("MaterialXTexture");
```

Comment:
This makes our nodes look more like other nodes in Hypershade (i.e. not including the "Node" suffix).